### ISSUE TITLE
Color-contrast rule returns incomplete and claims node is partially obscured

### ISSUE THREAD

**Issue by straker** [dequelabs/axe-core#1539]
Run Axe on https://straker.github.io/kontra/.

Expectation: Color contrast should pass

Actual: Color contrast returns incomplete for all nodes under the fold claiming the elements are partially obscured, even though no element obscures it.

I tried to figure out why the rule fails with @scurker, but we both gave up after we couldn't reproduce a reliable test case that worked consistently. 

The best we could determine is that when the color-contrast rule tries to determine the background color of the element, it thinks `document.body` does not visually contain the element. We thought it was because there is an inner scroll element `<main>` that captures the scroll instead of the body itself, so body never scrolls. But any time we tried to create a test case to reproduce that, the color-contrast rule would pass.

In summary: we think the code for `color.getBackgroundColor` scrolling the element with `scrollIntoView` is not scrolling the body element, so when checking `dom.visuallyContains`, the element fails for `document.body`.

<pre><code>
axe-core version: 3.2.2
axe-devtools 3.8.0
browser: Chrome
</code></pre>

**Comment by robertvrabel** [comment]
I'm having a similar issue. It looks like when putting a background-color on the body element it throws off the color-contrast check.

If you move it to the html element instead it seems to resolve the issue.

**Comment by jeankaplansky** [comment]
ID'd as color-contrast issue. We're going to make one entry into the release notes for all color-contrast fixes/enhancements.

**Comment by jeankaplansky** [comment]
ID'd as color-contrast issue. We're going to make one entry into the release notes for all color-contrast fixes/enhancements.

**Comment by padmavemulapati** [comment]
@straker , ran axe on, https://straker.github.io/kontra/ , I am seeing "all issues (17)" count, and all 17 were under "Needs to review" section. Verified with the latest axe. What is the expected behaviour here?

**Comment by padmavemulapati** [comment]
Now with the latest axe, getting all issues(17) in "Incomplete", working as expected.

**Comment by padmavemulapati** [comment]
![image](https://user-images.githubusercontent.com/47409579/74318327-052c0380-4da3-11ea-899c-c7e3fe43851f.png)
